lib/OpenQA/Script/CloneJob.pm

```diff
@@ -241,10 +241,37 @@ sub clone_jobs ($jobid, $options) {
     handle_tx($tx, $url_handler, $options, $jobs) if $tx;
 }
 
+sub detect_maintaince_update($job) {
+    my %settings = %{$job->{settings}};
+    my %test_template = ('BASE_TEST_TEMPLATE' => 'BASE_TEST_ISSUES', 'DESKTOP_TEST_TEMPLATE' => 'DESKTOP_TEST_ISSUES', 'SDK_TEST_TEMPLATE' => 'SDK_TEST_ISSUES', 'WE_TEST_TEMPLATE' => 'WE_TEST_ISSUES');
+    my @list_url ;
+    for ( keys %test_template) {
+       next  unless ($settings{$_} or  $settings{$test_template{$_}});
+       my $test_template = $settings{$_};
+       my $ua = LWP::UserAgent->new;
```

Comment:
I do not think there is a need to create a new UA for each cycle in the loop. I would move L251 and L252 to somewhere between L245 and L247.

Reply:
Yes, that’s a good idea. I have fixed it now.